packages/frontend/src/pages/EventDetailsPage/EventDetailsPage.tsx

```diff
@@ -0,0 +1,309 @@
+                      <path d="M7.8 10a2.2 2.2 0 004.4 0 2.2 2.2 0 00-4.4 0z"></path>
+                    </svg>
+                    {translation.completed}
+                  </span>
+                ) : (
+                  ''
+                )}
+                <div className={css.recruitmentWrapper}>
+                  <span className={css.recruitmentStatus}>Набор открыт</span>
+                </div>
+                <div className={css.participantsWrapper}>
+                  <span className={css.participants}>
+                    <svg
+                      xmlns="http://www.w3.org/2000/svg"
+                      fill="none"
+                      viewBox="0 0 24 24"
+                      stroke="currentColor"
+                      className={css.participantsIcon}
+                    >
+                      <path
+                        strokeLinecap="round"
+                        strokeLinejoin="round"
+                        d="M15 19.128a9.38 9.38 0 0 0 2.625.372 9.337 9.337 0 0 0 4.121-.952 4.125 4.125 0 0 0-7.533-2.493M15 19.128v-.003c0-1.113-.285-2.16-.786-3.07M15 19.128v.106A12.318 12.318 0 0 1 8.624 21c-2.331 0-4.512-.645-6.374-1.766l-.001-.109a6.375 6.375 0 0 1 11.964-3.07M12 6.375a3.375 3.375 0 1 1-6.75 0 3.375 3.375 0 0 1 6.75 0Zm8.25 2.25a2.625 2.625 0 1 1-5.25 0 2.625 2.625 0 0 1 5.25 0Z"
+                      />
+                    </svg>
+                    <span className={css.newApplications}>0</span>
+                    <span style={{ color: '#6B6888' }}>/</span>
+                    <span className={css.acceptedApplications}>0</span>
+                    <span style={{ color: '#6B6888' }}>/</span>
+                    <span className={css.participantsMaxNumber}>{event.volunteersQuantity}</span>
+                  </span>
+                </div>
+              </div>
+            )}
+          </div>
+          <div className={css.eventTitleWrapper}>
+            <span className={css.eventTitle}>{getEventNameInLanguage(event)}</span>
+          </div>
+          <div className={css.paricipantsTgChatWrapper}>
+            {event.status !== 'draft' ? (
+              <button
+                onClick={() => navigate(VOLUNTEERS_ROUTE)}
+                className={css.paricipantsTgChatButtons}
+              >
+                Список участников
+              </button>
+            ) : (
+              <button disabled className={css.paricipantsTgChatButtons}></button>
+            )}
+            {event.telegramChannelLink ? (
+              <a
+                href={event.telegramChannelLink}
+                target="_blank"
+                className={css.paricipantsTgChatButtons}
+              >
+                TG чат
+              </a>
+            ) : (
+              <a aria-disabled className={css.paricipantsTgChatButtons}>
+                TG чат
+              </a>
+            )}
+          </div>
+          <div className={css.wrapperForInfoBlock}>
+            <div>
+              <svg
```

Comment:
Could you save icon svgs from this file as separate files, please? It will help with cashing them and also they most likely will be reused